app/uk/gov/hmrc/emcstfe/repositories/CreateMovementUserAnswersRepository.scala

```diff
@@ -159,6 +166,91 @@ class CreateMovementUserAnswersRepositoryImpl @Inject()(mongoComponent: MongoCom
       )
       .toFuture()
       .map(_ => true)
+
+  def searchDrafts(ern: String, searchOptions: GetDraftMovementSearchOptions): Future[SearchDraftMovementsResponse] = {
+
+    val findFilter = {
+      Filters.and(
+        Seq(
+          //Drafts for this ERN
+          Some(Filters.equal(ernField, ern)),
+
+          //Drafts which match any search text across LRN, Consignee Name, Consignee ERN or Destination Warehouse ERN
```

Comment:
I don't think this `matches` it's more of a contains check

Reply:
Yeah agree, I will update the comment :)